config/routes.rb

```diff
@@ -15,16 +15,13 @@
   end
 
   devise_for :users,
-    controllers: { omniauth_callbacks: 'omniauth_callbacks', passwords: 'passwords' },
-    skip: [ :sessions, :registrations ]
+    controllers: { omniauth_callbacks: 'omniauth_callbacks', passwords: 'passwords', registrations:  'registrations' },
```

Comment:
Pretty sure we don't want to include all the devise registration controller functionality (specifically destroy) and instead just add custom routes to the `as :user` block to wire up to the registrations controller actions.